openapi/single-sign-on/single-sign-on.yaml

```diff
@@ -0,0 +1,384 @@
+          required: true
+          description: The Open ID authorization response type. Only 'code' is supported.
+        - schema:
+            type: string
+          in: query
+          name: code_challenge
+          description: 'OAuth2 PKCE Code Challenge generated just before making auth request, to be later verified at Token endpoint. Required when not using client secret.'
+        - schema:
+            type: string
+            enum:
+              - S256
+              - plain
+            example: S256
+          in: query
+          name: code_challenge_method
+          description: OAuth2 PKCE Code Challenge Method. Required when providing `code_challenge`. See PKCE documentation for details on these methods.
+        - schema:
+            type: string
+            example: openid
+          in: query
+          name: scope
+          required: true
+          description: 'OAuth2 Scopes being requested. Scopes are used to obtain access to User information and ability to call APIs on their behalf. Consent to scopes can be given and revoked, and requests for only consented scopes will not require interaction for consent unless `prompt=consent` forces it.'
+        - schema:
+            type: string
+          in: query
+          name: nonce
+          description: OpenID Nonce value to be returned in Identity Token. Optional on all supported flows.
+        - schema:
+            type: string
+          in: query
+          name: state
+          description: 'OAuth2 State. Opaque value used to maintain state between the request and the callback. Typically, Cross-Site Request Forgery (CSRF, XSRF) mitigation is done by cryptographically binding the value of this parameter with a browser cookie.'
+      tags:
+        - Single-Sign-On
+    parameters:
+      - schema:
+          type: string
+        name: partnerID
+        in: path
+        required: true
+        description: Partner ID providing identity namespacing
+  '/{partnerID}/oauth2/logout':
+    get:
+      summary: OpenID Logout
+      operationId: openid-logout
+      responses:
+        '200':
+          description: OK
+      x-lifecycle:
+        status: trustedTester
+      description: |-
+        [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Trusted Tester`
+
+        Browsers can be directed to this endpoint to logout. This will end the User's Session at Single-Sign-On and Sessions in other services which were accessed using the Single-Sign-On Session.
+
+        This operation has no security necessarily, but logout can only act upon a browser's session via the request's `vig_session` cookie.
+      parameters:
+        - schema:
+            type: string
+          in: query
+          name: post_logout_redirect_uri
+          description: 'URI to redirect to after logout. [This must be preregistered (spec), but we do not yet require or allow this]. [spec miss: id_token_hint is required when this parameter is used.]'
```

Comment:
```suggestion
          description: 'URI to redirect to after logout.
```
I dont think we can say anything more than this yet. Here is the story for the spec miss: https://vendasta.jira.com/jira/software/projects/INF/boards/187/backlog?selectedIssue=INF-1971